gammapy/background/models.py

```diff
@@ -71,3 +77,491 @@ def evaluate(self, x, y):
         y = np.asanyarray(y, dtype=float)
         parvals = self.parvals(x)
         return self._evaluate_y(y, parvals)
+
+
+def _make_bin_edges_array(lo, hi):
+    """Make bin edges array from a low values and a high values array.
+
+    TODO: move this function to somewhere else? (i.e. utils?)
+
+    Parameters
+    ----------
+    lo : `~numpy.array`
+    	lower boundaries
+    hi : `~numpy.array`
+    	higher boundaries
+
+    Returns
+    -------
+    bin_edges : `~numpy.array`
+    	array of bin edges as [[low], [high]]
+    """
+    return np.append(lo.flatten(), hi.flatten()[-1:])
+
+
+class CubeBackgroundModel(object):
+    """Cube background model.
+
+    TODO: this is a prototype that can only read and plot a cube.
+
+    Parameters
+    ----------
+    det_bins : `~astropy.coordinates.Angle`
+        Spatial bin edges vector (low and high). X, Y are equivalent.
+    energy_bins : `~astropy.units.Quantity`
+        Energy bin edges vector (low and high).
+    background : `~astropy.units.Quantity`
+    	Background cube in (energy, X, Y) format.
+    """
+
+    def __init__(self, det_bins, energy_bins, background):
+        self.det_bins = det_bins
+        self.energy_bins = energy_bins
+
+        # TODO: what's the axes order?
+        # ENERGY, DETX, DETY
+        # or
+        # ENERGY, DETY, DETX
+        self.background = background
+
+    @staticmethod
+    def read(filename):
+        """Read cube background model from fits file.
+
+        Parameters
+        ----------
+        filename : `~string`
+        	name of file with the bg cube
+
+        Returns
+        -------
+        bg_cube : `~gammapy.models.CubeBackgroundModel
+		bg model cube object
+        """
+        # TODO: should we use the Table class here to read the data?
+        hdu_list = fits.open(filename)
+        hdu = hdu_list['BACKGROUND']
+
+        header = hdu.header
+        data = hdu.data
+
+        det_bins_x = _make_bin_edges_array(data['DETX_LO'], data['DETX_HI'])
+        det_bins_y = _make_bin_edges_array(data['DETX_LO'], data['DETX_HI'])
```

Comment:
OK, I fixed the typo. For  storing `DETX` and `DETY` separatedly: I put it in my TODO list (I'll do it within this PR).